src/OceanSeaIceModels/InterfaceComputations/roughness_lengths.jl

```diff
@@ -28,15 +28,16 @@ end
     WindDependentWaveFormulation(FT = Oceananigans.defaults.FloatType;
                                  Umax = 19, ℂ₁ = 0.0017, ℂ₂ = -0.005)
 
-A gravity wave parameter based on the wind speed `ΔU` with the formula `ℂ₁ * max(ΔU, Umax) + ℂ₂`.
+A gravity wave parameter based on the wind speed `ΔU` with the formula `max(αmin, ℂ₁ * min(ΔU, Umax) + ℂ₂`)
+as shown in equation (13) of (Edson (2013)[@cite Edson2013].
```

Comment:
```suggestion
A gravity wave parameter based on the wind speed `ΔU` with the formula `ℂ₁ * max(ΔU, Umax) + ℂ₂`
as shown in equation (13) and described in surrounding text of (Edson (2013)[@cite Edson2013].
```

<img width="908" height="376" alt="image" src="https://github.com/user-attachments/assets/26d321ec-c435-4759-913c-d87b372b62c7" />

Reply:
might be worth commenting on Umax

Reply:
Also, I deleted the duplicate bibtex entry. There is already a bibtex entry for this paper; just change to `(@cite edson2013exchange)`.